### DISCUSSION TITLE
[Layouts] New App directory

### DISCUSSION THREAD

**Discussion by hosmelq** [vercel/next.js#39308] [Help] [answered]
I'm using the new `experimental.appDir` config, everything works fine locally during development and after build. But when I deploy to vercel I can not access pages inside the `app` directory.

**Comment by leerob** [comment, marked as answer]
**Update:** We’re excited to announce [Next.js 13](https://nextjs.org/13), which includes the new `app/` directory (beta) with support for nested layouts, colocation of data fetching, streaming, Server Components by default, and much more.

Inside the `app/` directory, there is a powerful new way to fetch data with React’s `use()` hook and the extended Web `fetch()` API. This allows you to colocate data fetching directly inside components for the most flexibility, including inside layouts.

You can leave feedback here: https://github.com/vercel/next.js/discussions/41745
